python/ray/_private/test_utils.py

```diff
@@ -1236,24 +1236,22 @@ def test_get_directory_size_bytes():
         assert ray._private.utils.get_directory_size_bytes(tmp_dir) == 152
 
 
-def check_local_files_gced(cluster):
+def check_local_files_gced(cluster, whitelist=None):
     for node in cluster.list_all_nodes():
         for subdir in ["conda", "pip", "working_dir_files", "py_modules_files"]:
             all_files = os.listdir(
                 os.path.join(node.get_runtime_env_dir_path(), subdir)
             )
             # Check that there are no files remaining except for .lock files
             # and generated requirements.txt files.
+            # Note: On Windows the top folder is not deleted as it is in use.
             # TODO(architkulkarni): these files should get cleaned up too!
-            if (
-                len(
-                    list(filter(lambda f: not f.endswith((".lock", ".txt")), all_files))
-                )
-                > 0
-            ):
-                print(str(all_files))
-                return False
+            items = list(filter(lambda f: not f.endswith((".lock", ".txt")), all_files))
 
+            if set(items) == whitelist:
```

Comment:
Based on the term "whitelist" I would expect subset instead of == here, what do you think?

Reply:
Yep, will update.